TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -99,6 +106,17 @@ def _get_basic_auth_password(self) -> str:
         password = client.get_secret_value(SecretId=self.auth_details["password_from_secret_arn"])
         return password["SecretString"]
 
+    def _get_sigv4_details(self, force_region=False) -> tuple[str, str]:
+        """Return the service signing name and region name. If force_region is true,
+        it will instantiate a boto3 session to guarantee that the region is not None.
+        This will fail if AWS credentials are not available.
+        """
+        assert self.auth_type == AuthMethod.SIGV4
+        if force_region:
```

Comment:
This is in preparation for the `CreateSnapshot` & `MigrateMetadata` commands, where region is a required parameter, so we'll want to get it no matter what.

For the cluster-specific calls, it's okay if we skip it because the auth object will figure it out itself.

Reply:
What do you think about using `boto3.session.Session().region_name` to resolve it up front?